_templates/component_v4/with-prompt/one-file.ejs.t

```diff
@@ -0,0 +1,30 @@
+---
+to: "<%= !multipleFiles ? `${path}/${tagName}/${tagName}.ts` : null %>"
+---
+import Component, { HTMLFragment, html, createStyle } from '@biotope/element';
+import * as styles from './<%= tagName %>.scss';
```

Comment:
Minor note: this will need to be changed for build v8 (to `import styles from ...`) - i.e. not now